reference/enterprise-api-ref/host.markdown

```diff
@@ -126,6 +126,39 @@ future. Key purging usually take an effect within 5-10 minutes.
 of [`trustkeysfrom`][cf-serverd#trustkeysfrom] for the hub hosts may re-appear
 and resume being collected from after being deleted.
 
+## Hosts list grouped by hard classes
+
+**URI:** https://hub.cfengine.com/api/hosts/by-class
+
+**Method:** GET
+
+**Parameters:**
+
+* **context-include** *(comma delimited string of regular expression strings)*
+    Includes hosts having context matching the expression.
+* **format** *(string)*
+    Output format. Default value is `json`. Allowed values: `json`, `yaml`.
+
+**Example response:**
```

Comment:
Can you please add in a CURL example call?

`curl -u USER:TOKEN https://hub/api/hosts/by-class?context-include=10_.*,127_.*,172_,ubuntu_.*`
?

Reply:
yes, sure.

Reply:
done